lib/web/desktop_playback.go

```diff
@@ -49,39 +49,45 @@ func (h *Handler) desktopPlaybackHandle(
 		return nil, trace.Wrap(err)
 	}
 
-	websocket.Handler(func(ws *websocket.Conn) {
-		ws.PayloadType = websocket.BinaryFrame
-
-		player, err := player.New(&player.Config{
-			Clock:     h.clock,
-			Log:       h.log,
-			SessionID: session.ID(sID),
-			Streamer:  clt,
-		})
-		if err != nil {
-			h.log.Errorf("couldn't create player for session %v: %v", sID, err)
-			ws.Write([]byte(`{"message": "error", "errorText": "Internal server error"}`))
-			return
-		}
+	upgrader := websocket.Upgrader{
+		ReadBufferSize:  4096,
+		WriteBufferSize: 4096,
+	}
+	ws, err := upgrader.Upgrade(w, r, nil)
```

Comment:
I'm not very familiar with this code, but WS after it's created with gorilla library should:
1. Set ping/pong handler, so the connection won't terminate because of timeout
2. Send WS 1006 message (WS connection closed) before closing the connection. I don't see it anywhere in this PR
3. Our implementation in other places sets some custom timeouts.

Reply:
1. I will look into whether ping/pong handler is required here. We didn't have any sort of ping happening with x/net/websocket so I suspect its fine.
2. This is completely optional and really only makes sense when you need graceful shutdown or when one side of the connection is going to shut down before the other. We use `ws.Close()` (which doesn't send a 1006) already in `lib/web/terminal` and `lib/web/assistant`.
3. I don't see any reason why that is necessary here.